doc/users/install.rst

```diff
@@ -14,7 +14,7 @@ You can follow the `Nipype tutorial <https://miykael.github.io/nipype_tutorial/>
 or use this docker container: `docker pull nipype/nipype`
 
 or if you want to build custom docker containers with specific versions of
-software see `Neurodocker <https://github.com/kaczmarj/neurodocker>`_
+software see Neurodocker_. Read the brief :doc:`neurodocker`.
```

Comment:
I don't like the way this whole section ends up looking, with blank lines followed by sentence fragments.

What about:

```RST
To get started using Docker, you can follow the `Nipype tutorial
<https://miykael.github.io/nipype_tutorial/>`_, or pull the `nipype/nipype`
image from Docker hub::

    docker pull nipype/nipype

You may also build custom docker containers with specific versions of software
using Neurodocker_ (see `Neurodocker tutorial`_).
```

(`:doc:` works, so go ahead and do that if you want. I'm just demonstrating what I meant earlier.)

Reply:
I added this, but the Neurodocker tutorial link still does not render properly for me when written this way. I kept the :doc: syntax.